docs_source_files/content/driver_idiosyncrasies/shared_capabilities.es.md

```diff
@@ -3,13 +3,7 @@ title: "Capacidades compartidas"
 weight: 1
 ---
 
-{{% notice info %}}
-<i class="fas fa-language"></i> Page being translated from 
-English to Spanish. Do you speak Spanish? Help us to translate
-it by sending us pull requests!
-{{% /notice %}}
-
-### pageLoadStrategy
+## pageLoadStrategy
 When navigating to a new page via URL, by default Selenium will wait
```

Comment:
There is still some text in English, can you please remove it?

Reply:
Was removed some paragraphs that still remained in English